examples/shared/src/test/scala/com/example/makro/ControllerTest.scala

```diff
@@ -33,18 +33,18 @@ class ControllerTest extends FunSuite with MockFactory {
  
     inSequence {
       inAnyOrder {
-        (mockTurtle.penUp _) expects ()
+        (() => mockTurtle.penUp) expects ()
```

Comment:
why has this style changed? Is this required by Scala 2.13?

Reply:
required by Scala 2.13. old style is compile error